examples/example09/rpm/harbour-demo.spec

```diff
@@ -0,0 +1,71 @@
+# 
+# Do NOT Edit the Auto-generated Part!
+# Generated by: spectacle version 0.27
+# 
+
+Name:       harbour-demo
+
+# >> macros
+# << macros
+
+%{!?qtc_qmake:%define qtc_qmake %qmake}
+%{!?qtc_qmake5:%define qtc_qmake5 %qmake5}
+%{!?qtc_make:%define qtc_make make}
+%{?qtc_builddir:%define _builddir %qtc_builddir}
```

Comment:
These shouldn't be here anymore. Use "Builder: qmake5" in the yaml to get rid of them.

Reply:
Thanks for this too, I've made this change and regenerated the spec files.